tests/state_test_util.go

```diff
@@ -218,8 +220,12 @@ func (t *StateTest) RunNoVerify(subtest StateSubtest, vmconfig vm.Config, snapsh
 	context := core.NewEVMBlockContext(block.Header(), nil, &t.json.Env.Coinbase)
 	context.GetHash = vmTestBlockHash
 	context.BaseFee = baseFee
+	if t.json.Env.Random != nil {
+		rnd := common.BigToHash(t.json.Env.Random)
+		context.Random = &rnd
+		context.Difficulty = big.NewInt(0)
```

Comment:
Why no `common.Big0` like below?

Reply:
Though, honestly I think it's better to leave as is and change the one below to new int. Thie big0 is a constant used in calculations and there we take care not to overwrite it, but setting in into a header field might be dangerous.